src/VisualStudio/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Roslyn.VisualStudio.Test.Utilities.Common;
+
+namespace Roslyn.VisualStudio.Test.Utilities.InProcess
+{
+    public class Editor_InProc : InProcComponent
```

Comment:
They're instance members because of how the infrastructure works. An `OutOfProcComponent` holds an instance of a type that derives from `InProcComponent`. If all of the members were static, the members couldn't be called on that instance, they'd have to be called off of `Editor_InProc`. Yes, it's technically possible to do that, but it breaks down the pattern I'm trying to establish about how code is called in the VS process vs. outside of it. Does that seem reasonable to you?